src/components/frame/v5/pages/UserCryptoToFiatPage/partials/ContactDetailsForm/validation.ts

```diff
@@ -1,9 +1,16 @@
 import * as Yup from 'yup';
 
+import { COUNTRIES_WITHOUT_STATES } from '~utils/countries.ts';
+
 export const addressValidationSchema = Yup.object({
+  country: Yup.string().required(),
   address1: Yup.string().required(),
   address2: Yup.string().notRequired(),
   city: Yup.string().required(),
-  state: Yup.string().required(),
+  state: Yup.string().when('country', {
+    is: (country) => COUNTRIES_WITHOUT_STATES.includes(country),
+    then: Yup.string().notRequired(),
+    otherwise: Yup.string().required(),
```

Comment:
This is more of a product question and better to do in a different PR but I wonder if we should also remove the State field if there are no options for it to begin with 🤔

Reply:
I agree that is a good idea for a future PR 👍